.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## What this PR does?
+
+<!-- Please describe what your changes does. -->
+
+## How to test your changes?
+
+<!-- Describe how we can test your changes? Is there any notable behaviour
+     that the end users could notice? -->
+
+## Related issue number
+
+<!-- Is there any issue opened that get fixed by this? -->
+
+## Checklist
+
+- [ ] Code is written
+- [ ] Tests are added
+- [ ] Documentation reflects your changes
+- [ ] CHANGES.txt is updated
```

Comment:
So there are two proposals; one is to collect CHANGES entries from the issue tracker, while the other is to have a separate file per CHANGES entry.